keps/sig-auth/20190711-external-credential-providers.md

```diff
@@ -0,0 +1,243 @@
+---
+title: External credential providers
+authors:
+  - "@awly"
+owning-sig: sig-auth
+participating-sigs:
+  - TBD
+reviewers:
+  - "@liggitt"
+  - "@mikedanese"
+approvers:
+  - "@liggitt"
+  - "@mikedanese"
+editor: TBD
+creation-date: 2019-07-11
+last-updated: 2019-07-11
+status: provisional
```

Comment:
this is retroactive and already implemented, mark as implementable

Reply:
Done